src/current/_includes/v24.3/wal-failover-metrics.md

```diff
@@ -0,0 +1,12 @@
+You can monitor if WAL failover occurs using the following metrics:
+
+- `storage.wal.failover.secondary.duration`: Cumulative time spent (in nanoseconds) writing to the secondary WAL directory. Only populated when WAL failover is configured.
+- `storage.wal.failover.primary.duration`: Cumulative time spent (in nanoseconds) writing to the primary WAL directory. Only populated when WAL failover is configured.
+- `storage.wal.failover.switch.count`: Count of the number of times WAL writing has switched from primary to secondary store, and vice versa.
+
+The `storage.wal.failover.secondary.duration` is the primary metric to monitor. You should expect this metric to be `0` unless a WAL failover occurs. If a WAL failover occurs, you probably care about how long it remains non-zero because it provides an indication of the health of the primary store.
```

Comment:
> you probably care about how long it remains non-zero because it provides an indication of the health of the primary store.

This metric is a counter, so once non-zero it will remain non-zero until the process restarts. The operator probably cares about the rate that it increases. Any increase indicates a failover event.